subsys/bluetooth/shell/bredr.c

```diff
@@ -531,6 +531,41 @@ static int cmd_sdp_find_record(const struct shell *sh,
 	return 0;
 }
 
+#if defined(CONFIG_BT_BONDABLE_PER_CONNECTION)
+static int cmd_conn_bondable(const struct shell *sh, size_t argc, char *argv[])
+{
+	int res;
+	const char *action;
+	bool enable;
+
+	if (!default_conn) {
+		shell_print(sh, "Not connected");
+		return 0;
+	}
+
+	action = argv[1];
+
+	if (!strcmp(action, "on")) {
+		enable = true;
+	} else if (!strcmp(action, "off")) {
+		enable = false;
+	} else {
+		shell_help(sh);
+		return 0;
+	}
+
+	shell_print(sh, "[%p] set conn bondable %s", default_conn, action);
+
+	res = bt_conn_set_bondable(default_conn, enable);
+	if (res) {
+		shell_error(sh, "Set conn bondable failed: result %d", res);
+		return -ENOEXEC;
+	}
+	shell_print(sh, "Set conn bondable done");
+	return 0;
+}
+#endif /* CONFIG_BT_BONDABLE_PER_CONNECTION */
```

Comment:
This isn't BR/EDR specific; should be in the `bt.c` file instead

Reply:
Done.